packages/trpc-interface/src/context/context.server.ts

```diff
@@ -23,11 +23,23 @@ type AuthorizationContext = {
   authorizeTrpc: TrpcInterfaceClient["authorize"];
 };
 
+type DomainContext = {
+  domainTrpc: TrpcInterfaceClient["domain"];
+  domainEnv: {
+    PUBLISHER_TOKEN: string;
+    PUBLISHER_ENDPOINT?: string;
```

Comment:
I wonder if this should ever be optional

Reply:
Im gonna remove it in next iterations. And use trpc for publish